src/PerfView/SupportFiles/UsersGuide.htm

```diff
@@ -4138,8 +4138,8 @@ <h3>Advanced Options</h3>
         </li>
         <li>
             The <strong><a id="AdditionalProvidersTextBox">Additional Providers TextBox</a></strong>
-            - A comma separated list of specifications for providers.&nbsp;&nbsp; This can be specified by using the
-            (the ... button) or by the following textual specification.  Each provider
+            - A comma separated list of specifications for providers cooresponding to the /providers comand line qualifier.
```

Comment:
typo: "command"

Reply:
I have fix this (actually in a number of places) as part of another checkin.